docker/rococo-node.dockerfile

```diff
@@ -0,0 +1,44 @@
+# Requires to run from repository root and to copy the binary in the build folder
+FROM --platform=linux/amd64 ubuntu:focal AS base
+LABEL maintainer="Frequency Team"
+LABEL description="Frequency parachain full node for Rococo testnet"
+
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+
+FROM --platform=linux/amd64 ubuntu:focal
+RUN useradd -m -u 1000 -U -s /bin/sh -d /frequency frequency && \
+	mkdir /data && \
+	chown -R frequency:frequency /data
+
+# RUN apt-get update && \
+# 	apt-get install -y jq curl tini
+
+USER frequency
+
+COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+# COPY --chown=frequency target/release/frequency.amd64 ./frequency/frequency
+COPY --chown=frequency target/release/frequency ./frequency/
+RUN chmod uog+x ./frequency/frequency
+
+# 9933 P2P port
+# 9944 for RPC call
+# 30333 for Websocket
+EXPOSE 9933 9944 30333
+
+VOLUME ["/data"]
+
+ENTRYPOINT ["/frequency/frequency", \
+	"--chain=frequency-rococo", \
+	"--force-authoring", \
+	"--base-path=/data", \
+	"--wasm-execution=compiled", \
+	"--execution=wasm", \
+	"--port=30333", \
+	"--rpc-port=9933", \
+	"--ws-port=9944", \
+	"--rpc-external", \
+	"--rpc-cors=all", \
+	"--ws-external", \
+	"--rpc-methods=Unsafe", \
```

Comment:
This should be `=safe`

(If you search substrate code for `check_if_safe` you will find all the rpcs that `=safe` will exclude)